components/introscreen/Introscreen.tsx

```diff
@@ -4,26 +4,24 @@ import Button from "../button/Button";
 
 function Introscreen() {
   return (
-    <>
-      <div className={styles.background}>
-        <h1 className={styles.headline}>Welcome to Australia</h1>
-        <div className={styles.box}>
-          <p>
-            Many animals live on this wonderful place and Albert,the platypus is
-            one of them.
-          </p>
-          <p>
-            Playing is very easy, just use your arrow keys or tap on the blue
-            Points to navigate Albert through the cities.
-          </p>
+    <div className={styles.background}>
+      <h1 className={styles.headline}>Welcome to Australia</h1>
+      <div className={styles.box}>
+        <p>
+          Many animals live on this wonderful place and Albert,the platypus is
+          one of them.
+        </p>
+        <p>
+          Playing is very easy, just use your arrow keys or tap on the blue
+          Points to navigate Albert through the cities.
+        </p>
 
-          <p> So lets meet Albert and discover the continent.</p>
-          <Button label="Traveling" />
-        </div>
-
-        <img className={styles.img} src="/australia-map.jpg" />
+        <p> So lets meet Albert and discover the continent.</p>
+        <Button label="Traveling" />
       </div>
-    </>
+
+      <img className={styles.img} src="/australia-map.jpg" />
```

Comment:
For better accessibility, it is best practice to include an alt attribute for images :)
```suggestion
      <img className={styles.img} src="/australia-map.jpg" alt="Map of Australia"/>
```